### ISSUE TITLE
Bundle Cable and Framed Type Cable

### ISSUE THREAD

**Issue by setsunafseiei** [age-series/ElectricalAge#144]
I would like to request bundle cable and framed type cable.
Bundle cable is easier to handle especially a multiwire setup
Framed type cable is easier depending on the build especially when it need to run between walls
similar design
https://projectredwiki.com/wiki/Bundled_Cable
https://projectredwiki.com/wiki/Framed_Bundled_Cable

**Comment by jrddunbr** [comment]
We have that, it's called SigBus.

**Comment by setsunafseiei** [comment]
> We have that, it's called SigBus.

i meant for Voltage or Current Cables

**Comment by jrddunbr** [comment]
*Someday...*

**Comment by jrddunbr** [comment]
Framed cables is probably something we'll do sooner or later. Probably after the pole update. By framed cables, you mean self-standing ones, right?

**Comment by setsunafseiei** [comment]
Yup which does not rely on blocks and i wish there is support for Forge Multipart (Multiblocks) support where wires can be hidden on walls

**Comment by jrddunbr** [comment]
Forge multipart is probably not happening. Not anytime soon, if it is even
possible. That said, covering blocks in Eln is possible by pressing C on a
cable and then right clicking a basic block on it.

The new framed cable would probably support the same thing

Jared

**Comment by setsunafseiei** [comment]
Thanks sir

**Comment by jrddunbr** [comment]
This issue is being closed now since there will be no more feature updates in Electrical Age for Minecraft 1.7.10.
If the issue is relevant to 1.12, it will be re-opened there at a later date.

**Comment by jrddunbr** [comment]
If we do FMP, I have a test in #369 which would be what it's built off.
